procurement_purchase_requisition_generation/wizards/product_replenish.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ProductReplenish(models.TransientModel):
+    _inherit = "product.replenish"
+
+    def _get_record_to_notify(self, date):
```

Comment:
Why do you need to override this? The usual use case flow is through MTO from sales order (anyway, this one should work also of course).